applications/luci-app-firewall/htdocs/luci-static/resources/view/firewall/zones.js

```diff
@@ -84,7 +84,9 @@ return L.view.extend({
 		o.modalonly = true;
 		o.cfgvalue = function(section_id) {
 			var name = uci.get('firewall', section_id, 'name');
-
+			if (name == null) {
```

Comment:
Please don't use curly brackets here if there is only one statement after the condition.